wolfcrypt/src/sp_int.c

```diff
@@ -4352,10 +4352,10 @@ static int _sp_mont_red(sp_int* a, sp_int* m, sp_int_digit mp);
  */
 static void _sp_zero(sp_int* a)
 {
-    a->used = 0;
-    a->dp[0] = 0;
+    ((sp_int_minimal *)a)->used = 0;
```

Comment:
Change this to: 
    sp_int_minimal* am = (sp_int_minimal*)a;

Reply:
done